sentry/templates/configmap-sentry.yaml

```diff
@@ -468,51 +406,76 @@ data:
     SENTRY_OPTIONS['mail.host'] = os.getenv("SENTRY_EMAIL_HOST", {{ .Values.mail.host | quote }})
     SENTRY_OPTIONS['mail.from'] = os.getenv("SENTRY_EMAIL_FROM", {{ .Values.mail.from | quote }})
 
+    #######################
```

Comment:
added for geo integration

[reference](https://github.com/getsentry/self-hosted/blob/master/sentry/sentry.conf.example.py#L283)